openedx/core/djangoapps/user_authn/views/tests/test_reset_password.py

```diff
@@ -606,6 +606,66 @@ def test_reset_password_with_other_user_link(self):
         self.assertRaises(Http404, PasswordResetConfirmWrapper.as_view(), reset_request, uidb36=self.uidb36,
                           token=self.token)
 
+    @ddt.data(
+        (None, None, True),
+        (None, 'invalid_token', False),
+    )
+    @ddt.unpack
+    def test_invalid_token_in_password_reset_request(self, uidb36, token, status):
```

Comment:
Shouldn't we create its own test class since it is a new endpoint and does not belong to `PasswordResetConfirmWrapper`?

Reply:
Done